freqtrade/exchange/exchange.py

```diff
@@ -1053,8 +1057,8 @@ def get_rate(self, pair: str, refresh: bool, side: str) -> float:
                     ticker_rate = ticker_rate - balance * (ticker_rate - ticker['last'])
             rate = ticker_rate
 
-        if rate is None:
-            raise PricingError(f"{name}-Rate for {pair} was empty.")
+        if rate is None and side == "sell":
+            raise PricingError(f"Sell-Rate for {pair} was empty.")
```

Comment:
Both buy and sell methods did raise a `PricingError` before - and can raise a pricing-error now as well.

As such - i really don't understand this change -n as it's changing behavior over the previous version.